docs/glossary.md

```diff
@@ -0,0 +1,361 @@
+# Glossary
+
+---
+
+## A
+---
+
+#### Account
+
+Accounts allow you to keep separate records of your DCR funds. Transferring DCR across accounts will create a transaction on the blockchain.
+
+#### Address
+
+A secure identifier that represents a possible destination for a payment, similar to an email address. The address is a human-readable representation of the public key receiving payment.
+
+#### Atom
+
+The smallest unit of Decred currency. One atom is one hundred millionth of a single DCR (0.00000001 DCR).
+
+
+## B
+
+---
+
+#### Block
+
+The data structure transactions are bundled into before being written into the distributed ledger. Decred's difficulty adjusts such that new blocks are found every five minutes on average.
+
+#### Block explorer
+
+A tool for inspecting the contents of blocks in a more user-friendly format. 
+
+#### Block header
+
+Metadata at the beginning of a block that defines information about the block. The hash of a block header is the main way of identifying a block. 
+
+#### Block reward
+
+A reward (in DCR) split between PoW miners, ticket holders, and the Treasury for successfully approving a block of transactions. The block reward [decays exponentially](https://docs.decred.org/advanced/inflation/) with block height (length of blockchain). However, Decred’s algorithm interpolates this decay over time so as not to produce market shocks, as typically seen in algorithms that involve large and infrequent reductions, such as halving the rewards every 4 years. The block reward started at 31.19582664 and it adjusts every 6,144 blocks (approximately 21.33 days) by reducing by a factor of 100/1011.
```

Comment:
> A reward (in DCR) split between PoW miners, ticket holders,

Instead of saying ticket holders here, maybe say voters.  Only voters who are picked for this block and whose votes appear in the block are rewarded.

> for successfully approving a block of transactions.

Both PoW miners and stake voters are rewarded even if the stake voters collectively vote to disapprove of regular transactions in the prior block.

Reply:
Have taken a stab at a new definition:

> A reward (in DCR) split between PoW miners, voters (ticket holders that have been called to vote), and the Treasury for successfully creating a block of transactions. 

For the second comment, I wasn't sure what language to use. I've just replaced 'approving' with 'creating', in hopes that works. After reading again the [hybrid-design](https://docs.decred.org/research/hybrid-design/) page, I have a better understanding. But the temporal logic still eludes me... Can you think of a better way to word this?

Reply:
Have gone with your suggested rewording. Fixed the reduction factor. Holding off for now on adding mention about possibly not getting block reward. This seems a little detailed for the glossary, and @RichardRed0x I believe is adding a mention or two about this in the big Jargon update PR: https://github.com/decred/dcrdocs/pull/590